draft-ietf-dprive-dnsoquic.md

```diff
@@ -294,9 +294,19 @@ This means that the client's first query occurs on QUIC stream 0, the second on
 When sending queries over a QUIC connection, the DNS Message ID MUST be set to
 zero.
 
-It is noted that this has implications for proxying DoQ message to other
-transports in that a mapping of some form must be performed (e.g., from DoQ
-connection/stream to unique Message ID).
+This has implications for proxying DoQ message to and from other transports. 
+
+One aspect is that a single DoQ connection can, in principle, support more
+outstanding queries at a given point in time than, for example, a single TCP
+connection which is limited by the Message ID space. Proxy implementations
+could choose to manage mappings where multiple TCP/DoT connections are required
+due to a large number of outstanding queries. For UDP, such a scenario will
+require other solutions e.g. delaying or dropping messages.
+
```

Comment:
Really? What kind of proxy are we speaking about? In the case of recursive resolvers, they will not forward all the traffic to a single connection. Some will be cached. Some will be fanned out to a variety of servers. Are you looking at something like a firewall?

Reply:
The text is correct, but I'm stuck deciding whether it is helpful.  It might be enough to just note that DoQ has a higher limit to the number of outstanding queries than other DNS transports.  Leaving the reader to discover the implications of that for their setting might be easier.

Reply:
I'm fine with that approach so I've pushed an update to trim this to be closer to original text.